gtk4-macros/src/lib.rs

```diff
@@ -5,8 +5,6 @@
 //! The crate aims to provide useful macros to use with the GTK 4 Rust bindings.
 
 mod attribute_parser;
-#[cfg(feature = "blueprint")]
-mod blueprint;
```

Comment:
Why this change?

Reply:
I can't export functions from gt4-macros. I'll revert this back 'cause I actually need to use a proc macro